docs/content/about/projects/_index.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Project Structure
+heading: OSCAL Project Structure
+weight: 20
+toc:
+  enabled: true
+---
+
+The OSCAL project is made up of several components, each of which lives in its own repository on GitHub under the [`usnistgov`](https://github.com/usnistgov) organization.
+
+These repositories form a complex patchwork of projects that depend on one-another:
+
```

Comment:
The Mermaid graph will need updates if you accept any of the suggestions related to the subtitles of the page.

Reply:
Resolved in https://github.com/usnistgov/OSCAL/pull/1702/commits/1a4914c840bfc5fd42ad873a2cd0c66bbf570c7a